### PR TITLE
[TIR] add roles fields and set correct columns in Relationships table

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/44226

## Description
- Add `Roles` field to all layouts
- Set all Relationships tables with the correct columns
## Minimum version of Cortex XSOAR
- [x] 6.5.0

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-ExecutiveBriefReport.json [hunk 1]
```diff
@@ -2,362 +2,378 @@
...
+            }
+        ]
+    },
     "name": "Executive Brief Report",
     "system": false,
     "version": -1,
-    "fromVersion": "6.5.0"
-}
+    "fromVersion": "6.5.0",
+    "description": ""
```
Comment (Itay4) [inline Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-ExecutiveBriefReport.json:378]: perhaps add description here?

File: Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-MalwareReport.json [hunk 2]
```diff
@@ -1,521 +1,537 @@
...
+                "readOnly": false,
+                "type": ""
+            }
+        ]
+    },
+    "name": "Malware Report",
+    "system": false,
+    "version": -1,
+    "fromVersion": "6.5.0",
+    "description": ""
```
Comment (Itay4) [inline Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-MalwareReport.json:536]: perhaps add description here?

File: Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-ThreatActorReport.json [hunk 3]
```diff
@@ -3,593 +3,609 @@
...
+            }
+        ]
+    },
     "name": "Threat Actor Report",
     "system": false,
     "version": -1,
-    "fromVersion": "6.5.0"
-}
+    "fromVersion": "6.5.0",
+    "description": ""
```
Comment (Itay4) [inline Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-ThreatActorReport.json:610]: perhaps add description here?

File: Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-VulnerabilityReport.json [hunk 4]
```diff
@@ -3,530 +3,546 @@
...
+            }
+        ]
+    },
     "name": "Vulnerability Report",
     "system": false,
     "version": -1,
-    "fromVersion": "6.5.0"
-}
+    "fromVersion": "6.5.0",
+    "description": ""
```
Comment (Itay4) [inline Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-VulnerabilityReport.json:547]: perhaps add description here?

File: Packs/ThreatIntelReports/ReleaseNotes/1_0_1.md [hunk 5]
```diff
@@ -5,7 +5,7 @@
 - Maintenance and stability enhancements.
 ##### Malware Report
 - Maintenance and stability enhancements.
-##### Campaign Report
+##_### Campaign Report
```
Comment (Itay4) [inline Packs/ThreatIntelReports/ReleaseNotes/1_0_1.md:8]: revert this

File: Packs/ThreatIntelReports/ReleaseNotes/1_0_3.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+##### Campaign Report
+- Added Roles field in status section and in new\edit form
+##### Vulnerability Report
+- Added Roles field in status section and in new\edit form
+##### Malware Report
+- Added Roles field in status section and in new\edit form
+##### Executive Brief Report
+- Added Roles field in status section and in new\edit form
+##### Threat Actor Report
+- Added Roles field in status section and in new\edit form
```
Comment (Itay4) [inline Packs/ThreatIntelReports/ReleaseNotes/1_0_3.md:12]:
```suggestion
- Added the *Roles* field.
##### Vulnerability Report
- Added the *Roles* field.
##### Malware Report
- Added the *Roles* field.
##### Executive Brief Report
- Added the *Roles* field.
##### Threat Actor Report
- Added the *Roles* field.
```

File: Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-VulnerabilityReport.json (hunk 4 above)
Comment (Ni-Knight) [reply Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-VulnerabilityReport.json]: Done

File: Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-MalwareReport.json (hunk 2 above)
Comment (Ni-Knight) [reply Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-MalwareReport.json]: Done

File: Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-ExecutiveBriefReport.json (hunk 1 above)
Comment (Ni-Knight) [reply Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-ExecutiveBriefReport.json]: Done

File: Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-CampaignReport.json [hunk 7]
```diff
@@ -3,448 +3,464 @@
...
+            }
+        ]
+    },
     "name": "Campaign Report",
     "system": false,
     "version": -1,
-    "fromVersion": "6.5.0"
-}
+    "fromVersion": "6.5.0",
+    "description": ""
```
[the remark this answers is not in the document]
Comment (Ni-Knight) [reply Packs/ThreatIntelReports/Layouts/layoutscontainer-ThreatIntelReport-CampaignReport.json]: Done

**Review by Itay4 — APPROVED** [review]

**Pull request merged**